code/lib/preview-api/src/modules/preview-web/PreviewWeb.tsx

```diff
@@ -7,5 +9,7 @@ import { WebView } from './WebView';
 export class PreviewWeb<TFramework extends Renderer> extends PreviewWithSelection<TFramework> {
   constructor() {
     super(new UrlStore(), new WebView());
+
+    global.__STORYBOOK_PREVIEW__ = this;
```

Comment:
Similarly, I think the caller should continue to do this rather than it doing itself